services/QuillLMS/client/app/bundles/Staff/components/evidence/configureRules/ruleHintPicker.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react";
+
+import RuleHintDropdown from './ruleHintDropdown';
+
+const RuleHintPicker = ({
+  hint,
+  onHintChange,
+}) => {
+
+  const hintId = (hint && hint.id) || null
+  const hintExplanation = (hint && hint.explanation) || null
+  const hintImageLink = (hint && hint.image_link) || null
+  const hintImageAltText = (hint && hint.image_alt_text) || null
```

Comment:
I think for all of these, the tidiest way to do it would just be:

`{ id, explanation, image_link, image_alt_text } = hint || {}`

You'd have to change the variable names below, but the file is so small I don't think you're at any risk of getting confused about what those attributes refer to.

Reply:
That seems reasonable.